src/unifiedreport/agent/reportStatic.php

```diff
@@ -113,6 +113,18 @@ function clearingProtocolChangeLogTable(Section $section)
     $section->addTextBreak();
   }
 
+  function checkValue($cell, $value, $text)
```

Comment:
I think the name of this method is confusing. My first association was something like "testValue"...
Maybe a name like "doCheckValue" or "addCheckbox" would be better...